flink-contrib/flink-connector-wikiedits/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
+			<artifactId>flink-clients_${scala.binary.version}</artifactId>
```

Comment:
It is executed in WikipediaEditsSourceTest. In the latest commit, this module still depends on flink-streaming-java, and also flink-clients of test scope.